src/operator/subgraph/dnnl/dnnl_fc_sum_fuse_property.h

```diff
@@ -95,42 +90,27 @@ class SgDNNLFCSumFuseSelector : public SubgraphSelectorV2 {
   bool SelectOutput(const BiDirectedNode& cur_node, const BiDirectedNode& output_node) override {
     const auto cur_n    = cur_node.node;
     const auto output_n = output_node.node;
-    if (status_ == kFail || status_ == kSuccess || output_n->is_variable()) {
+    if (status_ != kStart || output_n->is_variable()) {
```

Comment:
The same state machine process was done on other operators. Evee it is not fully utilized here yet, I think it is better to do it in the common way. (status_ == kFail || status_ == kSuccess ||)

Reply:
In my opinion it is a basic change and there is no need to keep this condition any longer than it needs to be especially that now it is way easier to read.